cif_pow.dic

```diff
@@ -4648,6 +4648,647 @@ save_pd_phase.name
+    _type.contents                Code
+
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         srefln                   'symmetric reflection'
+         arefln                   'asymmetric reflection'
+         strans                   'symmetric transmission'
+         atrans                   'asymmetric transmission'
+         cap                      'capillary (Debye-Scherrer)'
+    _enumeration.default          srefln
+
+save_
+
+save_pd_pref_orient.sphericalharmonics_id
+
+    _definition.id                '_pd_pref_orient.sphericalharmonics_id'
+    _definition.update            2022-09-28
+    _description.text
+;
+    A code to uniquely identify each spherical harmonic
+    preferred-orientation correction.
+;
+    _name.category_id             pd_pref_orient
+    _name.object_id               sphericalharmonics_id
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+    _enumeration.default          .
+
+save_
+
+save_pd_pref_orient.sphericalharmonics_phase_block_id
+
+    _definition.id
+        '_pd_pref_orient.sphericalharmonics_phase_block_id'
+    _definition.update            2022-09-28
+    _description.text
+;
+    A block ID code identifying the phase to which the
+    preferred-orientation correction applies. The data
+    block containing the crystallographic information
+    for this phase will be identified with a _pd_block.id
+    code matching the code in
+    _pd_pref_orient.sphericalharmonics_phase_block_id.
+;
+    _name.category_id             pd_pref_orient
+    _name.object_id               sphericalharmonics_phase_block_id
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_pd_pref_orient.sphericalharmonics_texture_index
+
+    _definition.id
+        '_pd_pref_orient.sphericalharmonics_texture_index'
+    _definition.update            2022-09-28
+    _description.text
+;
+    The texture index, as described by equation 4.212
+    in Bunge, as T = sum_{i,j} [c_{i,j)^2 / (2i+1)].
+
+    It gives the value [1, infty), where 1 is a random powder
+    and infty is an ideal single crystal.
+
+    Bunge, H.J., (2015) "Texture Analysis in Materials Science:
+    Mathemical Methods", Helga and Hans-Peter Bunge,
+    Wolfratshausen.
+;
+    _name.category_id             pd_pref_orient
+    _name.object_id               sphericalharmonics_texture_index
+    _type.purpose                 Measurand
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            1.0:
+    _units.code                   none
+    _method.purpose               Evaluation
+    _method.expression
+;
+    t = 0.0
+
+    Loop po as pd_pref_orient  {
```

Comment:
I'm probably not specifying it correctly, but this loop should only be done over c_ij/y_i values that belong to the same phase and histogram.

Reply:
The dREL can be written to automatically behave like this as long as the category has key data names that are linked to histogram id and phase id. Another reason to have them!